.github/workflows/facilitator-test.yml

```diff
@@ -0,0 +1,46 @@
+name: facilitator-test
```

Comment:
We already have build, test and clippy in the `build-rust` job in `.github/worksflows/ci-build.yml`. I think the thing to do is add the new coverage target there.